boltdesignsystem/bolt
# Testing | Applitools VRT | Base utils and accordion

## Jira



## Summary

This PR adds an integration with Applitools to Nightwatch.js.

## Details

There is a new Applitools class (singleton pattern to instantiate once). This can be imported into any nightwatch script. By running the function 
```
applitools.vrtTest({
      browser,
      testName: 'bolt-accordion',
      testingPath:
        '/pattern-lab/patterns/02-components-accordion-40-accordion-content-variations/02-components-accordion-40-accordion-content-variations.html',
    });
```
Nightwatch will take a full screenshot of the given url and upload that screenshot to Applitools.

## How to test

Log in to Applitools using Nedra's credentials. You should see multiple results for the Accordion component in different environments.
